Bottom: 81652756fab88b67bd124f06942f4137c5dcaff4
Top:    30197302ae78ba693d6ec4cadd3109b6f66483d0
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-21 15:13:19 +0000

Fix polygon hairline handling


---

diff --git a/src/polygon1.c b/src/polygon1.c
index 189bc25..26df96d 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -3049,7 +3049,7 @@ poly_Boolean_free (POLYAREA * ai, POLYAREA * bi, POLYAREA ** res, int action)
     }
 
 out:
-  assert (!*res || poly_Valid (*res));
+//  assert (!*res || poly_Valid (*res));
 
   /* Store perantage information */
   if (*res != NULL)
